### PR TITLE
Use selenium as main tool for e2e tests

### PR DESCRIPTION
**Pull request by maratfakhreev**

### CODE REVIEW

**Comment by ArturMinnullin** [comment]
:+1:

File: specs/features/landing_feature.js [hunk 1]
```diff
@@ -1,31 +1,48 @@
...
+    browser.findElement(selenium.By.name('password')).sendKeys('123456');
+    browser.findElement(selenium.By.css('#login_form_region button')).click();
+
+    browser.wait(() => {
+      return browser.isElementPresent(selenium.By.tagName('h2'));
+    }, setup.timeout);
 
-      this.waitForUrl(/dashboard/, function() {
-        expect('h2').to.contain.text('Greeting Gavin Jayson, you have next articles');
+    browser.findElement(selenium.By.tagName('h2')).then((el) => {
```
Comment (VladimirMikhailov) [inline specs/features/landing_feature.js]:
```js
browser.findElement(selenium.By.tagName('h2')).then((el) => {
```
just wondering, should the such construction automatically wait for a specific tag appearing on the page?

can we skip in this case?

```diff
+ browser.wait(() => {
+   return browser.isElementPresent(selenium.By.tagName('h2'));
+ }, setup.timeout);
```

Comment (maratfakhreev) [inline specs/features/landing_feature.js]: Yeah, it still works without `wait` part, I will update it. Thank you for your comment)

**Comment by VladimirMikhailov** [comment]
:gem: :+1:

**Comment by AmirMingaliev** [comment]
:ship:

File: specs/application_spec.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import Marionette from 'backbone.marionette';
 import App from 'scripts/application';
 
-describe('Application', function() {
-  it('should start successfully', function() {
+describe('Application', () => {
+  it('should start successfully', () => {
```
Comment (dmitrytrager) [inline specs/application_spec.js]: `it ('starts successfully', ...`

File: specs/models/article_spec.js [hunk 3]
```diff
@@ -12,23 +12,23 @@ describe('Article', function() {
     this.saveStub = sinon.stub(this.article, 'save');
   });
 
-  afterEach(function() {
+  afterEach(() => {
     this.saveStub.restore();
   });
 
-  it('article text should eq to "text"', function() {
+  it('article text should eq to "text"', () => {
```
Comment (dmitrytrager) [inline specs/models/article_spec.js]: same thing about 'should'

**Comment by dmitrytrager** [comment]
:+1:

**Pull request merged**